### PR TITLE
Tweak formatting config for exception lists

### PR DESCRIPTION
**Pull request by pugnascotia**
Change how we format exceptions to only wrap them as necessary. While
the config's overall philosophy is to put items one-per-line when
wrapping, in practice this is a little cumbersome for exception lists.

### CODE REVIEW

File: buildSrc/src/main/java/org/elasticsearch/gradle/precommit/DependencyLicensesTask.java [hunk 1]
```diff
@@ -210,8 +210,7 @@ private void failIfAnyMissing(String item, Boolean exists, String type) {
     }
 
     private void checkDependencies(Map<String, Boolean> licenses, Map<String, Boolean> notices, Set<File> shaFiles)
-        throws NoSuchAlgorithmException,
-        IOException {
+        throws NoSuchAlgorithmException, IOException {
```
Comment (original-brownbear) [inline buildSrc/src/main/java/org/elasticsearch/gradle/precommit/DependencyLicensesTask.java:213]: Just a question: I wonder if we want to give it another indent here for the `throws` if the `)` is not on the same line as the `throws` so it's one deeper than the method body? Is that possible?

**Review by original-brownbear — APPROVED** [review]
LGTM Thanks Rory!

File: buildSrc/src/main/java/org/elasticsearch/gradle/precommit/DependencyLicensesTask.java (hunk 1 above)
Comment (pugnascotia) [reply buildSrc/src/main/java/org/elasticsearch/gradle/precommit/DependencyLicensesTask.java:213]: I had another fiddle with the settings but this doesn't seem to be possible.

**Comment by pugnascotia** [comment]
@elasticmachine update branch

**Comment by pugnascotia** [comment]
Backported to `7.x` in 83e900e2483

**Pull request merged**